modules/file-extension-common/src/test/java/org/mule/runtime/module/extension/file/FilePredicateBuilderContractTestCase.java

```diff
@@ -269,6 +275,20 @@ public void rejectSymbolicLink()
         assertReject();
     }
 
+    @Test
+    public void failOnInvalidMinSize()
+    {
+        expectedException.expect(IllegalArgumentException.class);
```

Comment:
I used expectedException rule because that's the way I saw it was used in all File connector test cases (even though the test didn't check the exception message).